var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -403,6 +415,9 @@ class PyTensorflow(Package, CudaPackage, ROCmPackage, PythonExtension):
     # and https://github.com/abseil/abseil-cpp/issues/1665
     patch("absl_neon.patch", when="@2.16.1: target=aarch64:")
 
+    patch("Find_ROCm_Components_Individiually.patch", when="@2.14.0-rocm-enhanced +rocm")
+    patch("set_jit_trueLT_false.patch", when="@2.14.0-rocm-enhanced +rocm")
```

Comment:
Have these patches been submitted upstream yet?

Reply:
No the patches have not been upstreamed yet. Currently the code assumes that all the ROCm components are in a central location like **/opt/rocm**. This patch finds and set's the path for each ROCm componently. Unfortunately, It doesn't seem like these changes will be upstreamed anytime soon.

Reply:
What is preventing them from being upstreamed?

Reply:
Ah, I guess these are patches to https://github.com/ROCm/tensorflow-upstream, not to https://github.com/tensorflow/tensorflow. I'm looking forward to the day that https://github.com/ROCm/tensorflow-upstream can be contributed to https://github.com/tensorflow/tensorflow so we no longer have to maintain separate builds for both!